### PR TITLE
Frontend work for the professor course view

### PR DESCRIPTION
**Pull request by fredy20501**
- Also updated the create section frontend to make updating a section functional

### CODE REVIEW

File: src/store.js [hunk 1]
```diff
@@ -173,6 +175,80 @@ const store = new Vuex.Store({
...
+        },
+        attendance_threshold: 3,
+        students: [
+          {name: 'Frederic Verret'},
+          {name: 'John F Kennedy'},
+          {name: 'This is a very long name'},
+          {name: 'This is a very long name. Even longer than the other one'},
+          {name: 'Peter Parker'},
+          {name: 'Algernon Prime'},
+          {name: 'Mister Horsey McHorseface'},
```
Comment (ravenmills3) [inline src/store.js:197]: love this name. I will name my first child this

File: src/views/ProfessorHome.vue [hunk 2]
```diff
@@ -16,6 +16,14 @@ export default {
     ...mapGetters([
       'getUser'
     ])
+  },
+
+  methods: {
+    createSection() {
```
Comment (ravenmills3) [inline src/views/ProfessorHome.vue:22]: I think a more appropriate function name would be goToCreateSection?

**Review by ravenmills3 — APPROVED** [review]

File: src/views/ProfessorHome.vue (hunk 2 above)
Comment (fredy20501) [reply src/views/ProfessorHome.vue:22]: Yes that does sound better since we're not actually creating a section but rather redirecting to the page.

**Comment by fredy20501** [comment]
Here's what the UI looks like:
image
image

**Review by JoaoPontes98 — APPROVED** [review]
Lots of awesome work!

**Pull request merged**